src/VerletList.cpp

```diff
@@ -84,25 +86,180 @@ namespace espressopp {
   
   void VerletList::rebuild()
   {
+    timer.reset();
+    real currTime = timer.getElapsedTime();
+
     //real cutVerlet = cut + getSystem() -> getSkin();
     cutVerlet = cut + getSystem() -> getSkin();
     cutsq = cutVerlet * cutVerlet;
     
     vlPairs.clear();
 
-    // add particles to adress zone
-    CellList cl = getSystem()->storage->getRealCells();
-    LOG4ESPP_DEBUG(theLogger, "local cell list size = " << cl.size());
-    for (CellListAllPairsIterator it(cl); it.isValid(); ++it) {
-      checkPair(*it->first, *it->second);
-      LOG4ESPP_DEBUG(theLogger, "checking particles " << it->first->id() << " and " << it->second->id());
+    if(useBuffers) {
+      rebuildUsingBuffers(exList.size(), useSOA);
     }
```

Comment:
Please, keep the formatting consistent
` } else {`

Reply:
Edited.